react-front-end/secrets.cafe/src/App.tsx

```diff
@@ -2,8 +2,10 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import HomePage from "./pages/HomePage";
 import DonatePage from "@/pages/DonatePage";
```

Comment:
Faulty imports, change: `"@/pages/DonatePage";` to  `@/pages/donatePage` and  `"@/pages/DevelopPage"` to `"@/pages/developPage"`

Otherwise fails to resolve import. 

Or just change the file names as we seem to start with upper-case letters in rest of the pages.

Reply:
![image](https://github.com/mk-nlp/secrets-cafe/assets/159119624/5166c240-fbd4-485b-9d6c-6d38b8a446bc)
In my VSCode it's changed...?
Lemme just figure it out.

Reply:
![image](https://github.com/mk-nlp/secrets-cafe/assets/159119624/b65359ef-9b75-4607-988b-7d1ad6a2e4e1)

In my VSCode it's changed...?
I don't know what's wrong